### PR TITLE
Add a user guide for using Chassis for WP Core development

### PR DESCRIPTION
**Pull request by kadamwhite**
This new guide is based on [this gist by @rmccue](https://gist.github.com/rmccue/8b6c03db879cf785b433e65c860806f3), which outlines some basic configuration used to run a development copy of WordPress within Chassis for use when working on core tickets. This seems like useful material to include in the Chassis website.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ associated tools first:
 $ sudo pip install sphinx sphinx-autobuild sphinx_rtd_theme livereload==2.2.0
 ```
 
-Then, to build:
+Then to build, run the `sphinx-build` within this documentation folder:
 
 ```bash
-$ sphinx-build -b dirhtml _build/dirhtml
+$ sphinx-build -b dirhtml . _build/dirhtml
```
Comment (kadamwhite) [inline docs/README.md]: The source directory argument was missing, causing the local docs build to fail

File: docs/index.rst [hunk 2]
```diff
@@ -40,6 +40,7 @@ User Guide
    intro
    quickstart
    config
+   guides
```
Comment (kadamwhite) [inline docs/index.rst]: I'm not familiar with Sphinx so I may have missed some other place the new page had to be registered, but this seemed to be sufficient when building the docs locally

File: docs/README.md [hunk 3]
```diff
@@ -12,10 +12,10 @@ associated tools first:
 $ sudo pip install sphinx sphinx-autobuild sphinx_rtd_theme livereload==2.2.0
```
Comment (BronsonQuick) [inline docs/README.md]: Hey @kadamwhite! Thanks so much for the PR for this! 💖

I've just been testing this on a new laptop which is always nice as I can find out what dependencies I'm missing!

Could you please change this line to:
`sudo -H pip install Sphinx sphinx-autobuild sphinx_rtd_theme imagesize livereload==2.2.0`

I had some file permisson issues and was missing `imagesize` on this new computer and also `sphinx-build` wasn't found but changing it to `Sphinx` seemed to fix that for me.

Comment (BronsonQuick) [inline docs/README.md]: Also, seeing this new laptop is so fresh and clean I didn't have pip on here so could you please add something like the following to the bottom of the readme?

## Troubleshooting

If you see the following output in your terminal:
```bash
sudo: pip: command not found
```
You'll need to install pip with the following command:
```bash
sudo easy_install pip
```

File: docs/README.md (hunk 1 above)
Comment (BronsonQuick) [inline docs/README.md]: Good catch!

File: docs/index.rst (hunk 2 above)
Comment (BronsonQuick) [inline docs/index.rst]: Spot on Sir! You nailed it with this! 🔨

**Comment by BronsonQuick** [comment]
I should also cross reference https://github.com/Chassis/Chassis/issues/378 seeing this PR will fix that issue! 🕺🏼

**Comment by kadamwhite** [comment]
@BronsonQuick Thanks for the review! I've made those changes. I'd totally missed #378, too, I hope this helps folks in the future

**Pull request merged**